python/labbox_ephys/import_to_workspace.py

```diff
@@ -5,6 +5,9 @@
 import kachery_p2p as kp
 import kachery as ka
 
+# Note: the functions in this file are deprecated. Replaced by le.load_workspace(...). See import examples
```

Comment:
# Note: the functions in this file are deprecated. Replaced by le.load_workspace(...). See import examples
# This file will hang around for a bit, but ultimately be deleted